setup.py

```diff
@@ -50,7 +50,8 @@
 ]
 
 install_requires = [
-    'mosaicml[libcloud,wandb,mlflow,oci,gcs]>=0.17.2,<0.18',
+    'mosaicml[libcloud,wandb,oci,gcs]>=0.17.2,<0.18',
```

Comment:
hmm so i guess the potential issue here is that we're out of sync with the mlflow version that composer requires?

Reply:
The proper thing to do is probably to update the mlflow version in composer. We are going to pin composer in ftapi, so that's possible without a release. wdyt?

Reply:
No, I think this is correct. LLM Foundry is using a feature of MLflow that requires a tighter pin than the pin in composer. Composer has no reason to make its pin tighter, because it does not rely on this new feature. We could of course instead port the entire feature into the MLflowLogger, but I think it makes more sense here.

Reply:
ah yeah you're right. I was thinking about a case where composer requires an incompatible pin, but this should be fine. And agree, it makes more sense here 🚀